homeassistant/components/notify/html5.py

```diff
@@ -60,6 +63,7 @@ def get_service(hass, config):
 
     hass.wsgi.register_view(
         HTML5PushRegistrationView(hass, registrations, json_path))
+    hass.wsgi.register_view(HTML5PushCallbackView(hass, json_path))
```

Comment:
Where are we keeping it in sync? I see `self.registrations` being updated inside the registration view but that's it. Updates inside the view wouldn't go back up the chain, right? Or do I misunderstand a fundamental part of Python lol